src/installer/installer.go

```diff
@@ -337,12 +336,18 @@ func (i *installer) waitForNetworkType(kc k8s_client.K8SClient) error {
 }
 
 func (i *installer) waitForControlPlane(ctx context.Context) error {
-	kc, err := i.kcBuilder(KubeconfigPathLoopBack, i.log)
+	err := i.ops.ReloadHostFile("/etc/resolv.conf")
+	if err != nil {
+		i.log.WithError(err).Error("Failed to reload resolv.conf")
+		return err
+	}
+	kc, err := i.kcBuilder(KubeconfigPath, i.log)
```

Comment:
so u basically pass this stage when there will be at least resolvable 1 api server ? Meaning at least 1 master.
But can it happen that it will move to another node later on, for exmaple in waitForMinMasterNodes - do u deal with the disruption in the kcBuilder? Or the assumption is the api is stable

Reply:
Not exactly, this means we don't care if the kube-apiserver is running on the bootstrap or one of the master nodes.
The disadvantage (compared to loopback kubeconfig) is that now we depend on coredns and keepalived in order to resolve the cluster DNS to the node running the kube-apiserver

Reply:
I guess this is the tradeoff.
What is more prone to failure:
1. Current implementation that depends on master nodes being ready before kube-apiserver move from the bootstrap.
2. This PR implementation that relay on keepalived and coredns for accessing the kube-apiserver.